variant_calling_benchmarks/common.py

```diff
@@ -28,7 +28,10 @@ def extract_loci_string(patient, variant_filenames):
 
 def add_common_run_args(parser):
     parser.add_argument("configs", nargs="+", help="JSON config files")
-    parser.add_argument("--guacamole-jar", required=True)
+    parser.add_argument("--guacamole-jar", required=True,
```

Comment:
Yeah, I was thinking about something like that. But then there's the quiet and error prone logic about picking which jar to use (i.e. the version stuff that `scripts/guacamole` does), which seems maybe better to just be explicit about